src/components/StarRating/StarRating.scss

```diff
@@ -0,0 +1,31 @@
+$color-placeholder-star: #e1e1e1;
+$color-active-star: #d9af00;
+$color-number-reviews: gray;
+
+.CoveoStarRating {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+}
+
+.coveo-star-rating-star + .coveo-star-rating-star {
+    margin-left: 2px;
+}
+
+.coveo-star-rating-star-svg {
+    width: 15px;
+    height: 15px;
+    color: $color-placeholder-star;
+}
+
+.coveo-star-rating-star-active {
+    .coveo-star-rating-star-svg {
+        color: $color-active-star;
+    }
+}
+
+.coveo-star-rating-label {
+    margin-left: 5px;
+    font-size: 11px;
+    color: $color-number-reviews;
+}
```

Comment:
You can simplify this by using extending a base `.coveo-star-rating` class like this:
`.coveo-star-rating-star {
   &-svg {
      stuff here
   }
   other stuff here
}`

Reply:
no worries I know what you're talking about. Like i said elsewhere, i just copied the css and changes the class names not wanting to change what isn't broke